include/deal.II/cgal/polygon_2.h

```diff
@@ -0,0 +1,139 @@
+#ifndef dealii_cgal_polygon_2_h
+#define dealii_cgal_polygon_2_h
```

Comment:
I think we should drop the _2 from the filename. Polygon already includes the information that those are 2D objects.